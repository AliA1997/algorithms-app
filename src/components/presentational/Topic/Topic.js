import React from 'react';

export const Topic = (props) => {
    return (
      <div>
          Topic
      </div>  
    );
};

export default Topic;