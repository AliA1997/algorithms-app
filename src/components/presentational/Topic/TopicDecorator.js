import React from 'react';
///import withHandlers from topicDecorator 
import { withHandlers } from 'recompose';



const handlerHOC = withHandlers({
})