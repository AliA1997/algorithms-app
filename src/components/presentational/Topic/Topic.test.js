//import react and component.
import React from 'react';
import { Topic } from './Topic';
//import the render to create a mock component.
import render from 'react-test-renderer';
//NOTE WHEN update snapshots use --updateSnapshot

//Check if the component renders correctly
describe('Check if the component renders correctly', () => {
    //Check if the component renders correctly
    test('Check if component renders correctly!', () => {
        //create your mock component.
        const tree = render.create(<Topic />).toJSON();
        expect(tree).toMatchSnapshot();
    })
})