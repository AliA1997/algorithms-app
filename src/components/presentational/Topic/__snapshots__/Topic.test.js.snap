// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check if the component renders correctly Check if component renders correctly! 1`] = `
<div>
  Topic
</div>
`;
